x-pack/modules/README.md

```diff
@@ -1,99 +0,0 @@
-# Module settings and structure
```

Comment:
given we're not yet removing all modules, maybe we can lead the READMEs intact for now